### PR TITLE
🌐 [i18n-KO] Translated `transformers_agents.md` to Korean

### PR DESCRIPTION
**Pull request by sim-so**
# What does this PR do?

Translated the `transformers_agents.md` file of the documentation to Korean.
Thank you in advance for your review. 

Part of https://github.com/huggingface/transformers/issues/20179

## Before reviewing
- [x] Check for missing / redundant translations (번역 누락/중복 검사)
- [x] Grammar Check (맞춤법 검사)
- [x] Review or Add new terms to glossary (용어 확인 및 추가)
- [x] Check Inline TOC (e.g. `[[lowercased-header]]`)
- [x] Check live-preview for gotchas (live-preview로 정상작동 확인)

## Who can review? (Initial)

May you please review this PR? 
@sronger, @TaeYupNoh, @kj021, @HanNayeoniee, @eenzeenee, @sim-so

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review? (Final)

@sgugger, @ArthurZucker, @eunseojo 
May you please review this PR?

### CODE REVIEW

File: docs/source/ko/transformers_agents.md [hunk 1]
```diff
@@ -0,0 +1,328 @@
...
+
+<img src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/transformers/sea_and_island.png" width=200>
+
+<br/>
+
+
+모든 [`~Agent.run`] 작업은 독립적이므로 다른 작업으로 여러 번 연속해서 실행할 수 있습니다.
+
+`agent`는 큰 언어 모델일 뿐이므로 프롬프트에 약간의 변화를 주면 완전히 다른 결과가 나올 수 있다는 점에 유의하세요. 
+수행하려는 작업을 가능한 한 명확하게 설명하는 것이 중요합니다.
```
Comment (sronger) [inline docs/source/ko/transformers_agents.md:150]:
```suggestion
수행하려는 작업을 최대한 명확하게 설명하는 것이 중요합니다. 
```
"가능한 한" 보다 "최대한"이 더 잘 읽히는 것 같아서 제안 드립니다 !

File: docs/source/ko/transformers_agents.md [hunk 2]
```diff
@@ -0,0 +1,328 @@
...
+agent.chat("Transform the picture so that there is a rock in there")
+```
+
+<img src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/transformers/rivers_and_lakes_and_beaver.png" width=200>
+
+<br/>
+
+이 방식은 여러 명령어에 걸쳐 상태를 유지하고자 할 때 흥미로운 접근 방식입니다. 
+실험용으로 더 좋지만 복잡한 명령어보다는 
+단일 명령어(`~Agent.run` 메소드가 더 잘 처리하는 명령어)에 훨씬 더 잘 작동하는 경향이 있습니다.
```
Comment (eenzeenee) [inline docs/source/ko/transformers_agents.md:203]:
```suggestion
단일 명령어([`~Agent.run`] 메소드가 더 잘 처리하는 명령어)에 훨씬 더 잘 작동하는 경향이 있습니다.
```

185번 줄의 ['~Agent.chat']과 비교하여 각괄호가 누락된 것 같아 추가해보았습니다.

**Comment by sim-so** [comment]
@eenzeenee @sronger님 리뷰 감사합니다! 
제안해주신 수정사항을 모두 반영했습니다 ☺️

**Comment by sim-so** [comment]
Could you review this PR? 😃
@sgugger, @ArthurZucker, @eunseojo

**Review by sgugger — APPROVED** [review]
Thanks!

**Pull request merged**